### PR TITLE
[Refactor] Abstract all common information fields into another sub-package

### LINKED ISSUE #59: Abstract all common information fields into another sub-package

### PR DESCRIPTION
**Pull request by KengHian**
- close #59 
- Just moving stuff around and renaming things, should be an easy one!

### CODE REVIEW

**Review by wassilililily — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/model/common/Name.java [hunk 1]
```diff
@@ -16,7 +16,7 @@ public class Name {
      * The first character of the address must not be a whitespace,
      * otherwise " " (a blank string) becomes a valid input.
      */
-    public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
+    public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum}, ]*";
```
Comment (Green-Tea-123) [inline src/main/java/seedu/address/model/common/Name.java:19]: Is this supposed to be the correct regex? The company name uses a different regex as it has different conditions

**Review by Green-Tea-123 — COMMENTED** [review]
I think we need to discuss the company name restrictions as it was the initial reason for having 2 different names

File: src/main/java/seedu/address/model/common/Name.java (hunk 1 above)
Comment (KengHian) [reply src/main/java/seedu/address/model/common/Name.java:19]: Alright let's discuss it tomorrow

File: src/main/java/seedu/address/logic/commands/AddJobCommand.java [hunk 2]
```diff
@@ -21,7 +21,7 @@ public class AddJobCommand extends AddCommand<Job> {
     public static final String FULL_COMMAND = COMMAND_WORD + " " + ENTITY_WORD;
     public static final String MESSAGE_USAGE = FULL_COMMAND + ": Adds a job listing to the address book. "
             + "Parameters: "
-            + PREFIX_NAME + "JOBNAME "
+            + PREFIX_NAME + "NAME "
```
Comment (AlekKwek18) [inline src/main/java/seedu/address/logic/commands/AddJobCommand.java:24]: LGTM. More consistent naming.

**Review by AlekKwek18 — APPROVED** [review]
Overall, LGTM

File: src/main/java/seedu/address/model/common/Name.java (hunk 1 above)
Comment (KengHian) [reply src/main/java/seedu/address/model/common/Name.java:19]: As discussed, considering alnum and comma will cover most of the use case, we will proceed the this simpler implementation for now.

**Pull request merged**